auxiliary/debug_web_cv2.py

```diff
@@ -0,0 +1,34 @@
+import threading
```

Comment:
Previously, this script was created to ensure that the result from the browser is the same as the result from the CV2. When we see that it's already working together, I thought that the script is no longer needed. 

 But now, after understanding that we may sometime need this script again, I think it would be nice to integrate it with the current `dot_vision.py`.